lib/src/open_prices_api_client.dart

```diff
@@ -63,6 +64,28 @@ class OpenPricesAPIClient {
         addUserAgentParameters: addUserAgentParameters,
       );
 
+  static Future<MaybeError<PriceUser>> getUserProfile(
+    final String userId,
```

Comment:
```suggestion
    final String username,
```
For consistency with the rest of the code.